migrations/0028_green_gamora.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "options" RENAME COLUMN "accepted" TO "show";--> statement-breakpoint
+ALTER TABLE "events" ADD COLUMN "fields" jsonb DEFAULT '[]'::jsonb NOT NULL;--> statement-breakpoint
+ALTER TABLE "registrations" ADD COLUMN "data" jsonb;--> statement-breakpoint
+ALTER TABLE "questions" ADD COLUMN "fields" jsonb DEFAULT '[]'::jsonb NOT NULL;--> statement-breakpoint
+ALTER TABLE "options" ADD COLUMN "data" jsonb;
```

Comment:
why two migrations instead of one? I.e. why not db:drop both and create only one related to the PR?

Reply:
they could be one, generally speaking i did it because of separation of concerns in case we have to go back a migration. but I think it could be one if it is approved

Reply:
ahh okey makes sense. Never mind then :)